### ISSUE TITLE
Error when installing dependencies: "Your Ruby version is 2.6.3, but your Gemfile specified 2.3.6"

### ISSUE THREAD

**Issue by jmjavier** [tofugu/wanikani-knowledgebase#52]
Installed ruby 2.6.3 with rvm for this repo.
However when running `bundle install` I get the error:
``` 
bundle install
Your Ruby version is 2.6.3, but your Gemfile specified 2.3.6
```

**Comment by jmjavier** [comment]
Oops. I got mixed up with the numbers there trying 2.3.6 now..

**Comment by vietqhoang** [comment]
Have you tried switch the ruby version with RVM?

```
rvm use 2.3.6
```

The `Gemfile` has the Ruby version set: https://github.com/tofugu/wanikani-knowledgebase/blob/master/Gemfile#L3. So if you open a new terminal to the project it should use the project's Ruby version. But if you just installed it and haven't refreshed your terminal, then a different Ruby version may be loaded and you'll need to either reload the Terminal window or manually switch to the Ruby version using rvm.

**Comment by jmjavier** [comment]
Could it be related to bundler versioning?
I did both rvm use and open a session and it does seem to set 2.3.6 
```
[javi] ~/Work  % cd wanikani-knowledgebase
RVM used your Gemfile for selecting Ruby, it is all fine - Heroku does that too,
you can ignore these warnings with 'rvm rvmrc warning ignore /Users/javi/Work/wanikani-knowledgebase/Gemfile'.
To ignore the warning for all files run 'rvm rvmrc warning ignore allGemfiles'.
[javi] ~/Work/wanikani-knowledgebase [master]  % rvm use
RVM used your Gemfile for selecting Ruby, it is all fine - Heroku does that too,
you can ignore these warnings with 'rvm rvmrc warning ignore ./Gemfile'.
To ignore the warning for all files run 'rvm rvmrc warning ignore allGemfiles'.

Using /Users/javi/.rvm/gems/ruby-2.3.6
[javi] ~/Work/wanikani-knowledgebase [master]  % bundle install
Your Ruby version is 2.6.3, but your Gemfile specified 2.3.6
```

and then tried
```
[javi] ~/Work/wanikani-knowledgebase [master]  % which ruby
/Users/javi/.rvm/rubies/ruby-2.3.6/bin/ruby
[javi] ~/Work/wanikani-knowledgebase [master]  % ruby -v
ruby 2.3.6p384 (2017-12-14 revision 61254) [x86_64-darwin20]
[javi] ~/Work/wanikani-knowledgebase [master]  % bundle install
Your Ruby version is 2.6.3, but your Gemfile specified 2.3.6
[javi] ~/Work/wanikani-knowledgebase [master]  % bundle -v
Bundler version 1.17.2
[javi] ~/Work/wanikani-knowledgebase [master]  % which bundle
/usr/bin/bundle
[javi] ~/Work/wanikani-knowledgebase [master]  % which bundler
/usr/bin/bundler
[javi] ~/Work/wanikani-knowledgebase [master]  % bundle install
Your Ruby version is 2.6.3, but your Gemfile specified 2.3.6
[javi] ~/Work/wanikani-knowledgebase [master]  % rvm use 2.3.6
Using /Users/javi/.rvm/gems/ruby-2.3.6
[javi] ~/Work/wanikani-knowledgebase [master]  % bundle install
Your Ruby version is 2.6.3, but your Gemfile specified 2.3.6
```

**Comment by vietqhoang** [comment]
@jmjavier Can you check and see https://github.com/tofugu/wanikani-knowledgebase/pull/53 addresses the issues you are experiencing?

**Comment by vietqhoang** [comment]
Sorry. I closed #53 in favor of #54. Please try out #54 to see if it address what you have been experiencing.
